nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/test-flows/NIFI_4827.xml

```diff
@@ -0,0 +1,635 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
It's a miniature test flow that's meant to show the behavior if someone has questions. I got the idea when I was reviewing the PutInfluxDb processor. On nifi-dev, I think @joewitt mentioned that a major problem with some of these bundles is the people who actually know a particular system might be highly limited in the community. I think getting into a habit of sharing test flows that use something like GenerateFlowFile could ease the learning curve on others. Particularly if a docker-compose file or something like that is added with it.

I can remove it, but I think it might be a good practice.

Reply:
I like the idea of having test flows, but maybe not in the codebase since they aren't executed? I often put test flows up as a Gist on GitHub and refer to it in the PR, so the reviewer can go there for a sample flow (which I try to document inline with labels to explain what's going on)

Reply:
If not there, it should be some place that is owned by the NiFi team so it can easily survive us if the infamous "hit by a bus" scenario happens.

Reply:
Agreed, I often make the Gist available for the reviewer, then once merged, the template could be added to the Wiki of sample templates for all to have

Reply:
Ok. Removed this for now since it can go on the Wiki.